hwstats/frontend.py

```diff
@@ -65,10 +66,12 @@ def process_plot(pid_hash: str) -> str:
 
     cpu_fig = get_time_plot_fig(pid_hash, session, query_cpu_percent_with_time)
     mem_fig = get_time_plot_fig(pid_hash, session, query_memory_percent_with_time)
+    disk_fig = get_time_plot_fig(pid_hash, session, query_Disk_percent_with_time)
```

Comment:
The `get_time_plot_fig` method expects a two element tuple so we really can't use it. You could build a new very similar method tho just to handle generating the disk plot with it's three elements